docs/source/redhat-opens-install.rst

```diff
@@ -111,6 +111,9 @@ please get in touch with us and we'll be happy to help you through the process.
 
    - Download, unpack, and install the binary::
 
+.. warning:: We've seen certificate errors downloading etcd - you may need to
+             add --insecure to the `curl` command to ignore this.
+
```

Comment:
This ends up with everything in the rest of step 1 in the warning box - its not immediately obvious to me how to fix this given the rest of the indenting, but maybe you should just not bother with the warning and add --insecure to the example command.